_data/glossary/configmap.yaml

```diff
@@ -0,0 +1,13 @@
+id: configmap
+name: ConfigMap
+full-link: /docs/tasks/configure-pod-container/configmap/
+related:
+- pod
+- secret
+tags:
+- core-object
+short-description: >
+  Used to store non-confidential data in key-value pairs and consumed as environment variables, command-line arguments, or config files in a {% glossary_tooltip text="volume" term_id="volume" %}.
+long-description: >
+  Allows you to decouple configuration artifacts from image content to keep containerized applications portable.
```

Comment:
Two suggestions:
* Consider using the custom tag `glossary_tooltip` (e.g. `{% glossary_tooltip text="Services" term_id="service" %}`, specifically for the term `Secret` once https://github.com/kubernetes/website/pull/6577 is merged).

* I like the way you approached this but "artifact" can be a hefty word. Here's a suggestion, although you can feel free to disagree:
```
Allows you to decouple environment-specific configuration from your 
{% glossary_tooltip text="container images" term_id="container" %}, 
so that your applications are easily portable.
```

Reply:
done